api/src/db/migrations/20240720091126-add-profile-in-users-table.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addColumn('users', 'profile', {
```

Comment:
update the name avatar instead of the profile

Reply:
avatar name is updated